.github/workflows/buildTestWindows.yml

```
@@ -0,0 +1,31 @@
+name: Java CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ main ]
+
+jobs:
+  testOnWindows:
+    runs-on: windows-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+          cache: maven
+      - name: Build with Maven
+        run: mvn --batch-mode --update-snapshots verify
```

```suggestion
        run: mvn --show-version --batch-mode --strict-checksums verify
```

Adding `--show-version` will let us see what Maven version runs here, to understand if that JavaDoc error is related to that (on Travis it's Apache Maven 3.8.6).

Removing `--update-snapshots` should't cause any issues, but be faster.

Adding `--strict-checksums` seems like a good idea, security wise.

I'm making [the Travis CI configuration](https://github.com/vorburger/MariaDB4j/blob/main/.travis.yml) similar with https://github.com/vorburger/MariaDB4j/commit/aeda8bea13e0f932e05a42bf24391e2b208cc9c4.